CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   did not have any files associated.
 - Ensured that digests are always added to built RPMs. Previously they would not be included unless
   the "signature-meta" (or "signature-pgp") features were enabled.
+- Move to opt-in for reproducability inhibiting by default values, `build_time` and `build_host`.
+- Removed `get_build_cookie` again
```

Comment:
I think we should keep it personally.  It's so simple there's limited reason not to, and makes it easy for users to do the standard thing even if they don't know exactly what the "standard thing" is, which is useful.

But for this or anything else it it would be better to just remove the changelog line where it was added.

Reply:
It adds the `gethostname` dependency and I see this as an Impl detail of the user of the library.

Reply:
Ok.  Well, the second points remains, just alter / remove the original changelog entry rather than adding a new one.